requirements.txt

```diff
@@ -1,5 +1,5 @@
-bittensor==5.2.0
-transformers<=4.28.0 
+bittensor>=5.3.2,<6.0.0
```

Comment:
Which one is bittensor version 6.0.0?

Reply:
`revolution`. Also just the next major, so we avoid breaking API changes